gitrevise/tui.py

```diff
@@ -230,7 +230,7 @@ def inner_main(args: Namespace, repo: Repository) -> None:
     if args.all:
         repo.git("add", "-u")
     if args.patch:
-        repo.git("add", "-p")
+        repo.git("add", "-p", nocapture=True)
```

Comment:
This was also fixed in this #99 (see discussion: https://github.com/mystor/git-revise/pull/99#discussion_r720942365).

That PR has been ready to merge for quite a while. (cc @mystor @Manishearth @krobelus).

Reply:
Good to see. Our fixes are effectively the same. They would probably be identical if you were to reorder your commits to put the fix first.

I would typically put the fix first, though (to minimize the bug's lifetime in commits), even as you said in your commit that the mistake becomes obvious after making the argument explicit, which I think is a right response. There is some truth to `python -c 'import this' | grep Explicit`. However, as soon as all callers are explicit, I think the argument needn't/shouldn't be optional anymore. And as a consideration, if it helps verbosity: The distinction can also be made obligatory by wrapper functions, such as `git_interactive()` vs `git_popen()` vs `git_silent()`.